src/main/kotlin/eu/europa/ec/eudi/openid4vci/CredentialConfiguration.kt

```diff
@@ -28,6 +28,13 @@ import java.util.*
  */
 sealed interface CryptographicBindingMethod : Serializable {
 
+    /**
```

Comment:
This is not in the specification Please remove it.

Reply:
Done